src/main/java/frc/robot/Robot.java

```diff
@@ -144,7 +186,7 @@ public void teleopInit() {
   @Override
   public void teleopPeriodic() {
     Scheduler.getInstance().run();
-
+    m_climber.updateState();
```

Comment:
This is the thing that tells the climber to check the choosers for talon settings.